frontend/packages/core/src/typography.tsx

```diff
@@ -62,9 +62,14 @@ const STYLE_MAP = {
     lineHeight: "22",
   },
   body3: {
+    size: "14",
+    weight: REGULAR,
+    lineHeight: "18",
+  },
+  body4: {
     size: "12",
     weight: REGULAR,
-    lineHeight: "20",
+    lineHeight: "16",
```

Comment:
some discrepancies with the core design system. `body3` was using `body4` styling and we were missing the `body4` variant